nixos/modules/services/security/wazuh/generate-agent-config.nix

```diff
@@ -0,0 +1,169 @@
+{ cfg, pkgs }:
+pkgs.writeText "ossec.conf" ''
```

Comment:
This file is pretty big and it would hard to keep it in sync with the upstream.
I suggest to use the upstream file and `substitute` a few values.

Reply:
Agreed, we could simplify this. I'll explore just using substitution.